watertap/unit_models/osmotically_assisted_reverse_osmosis_base.py

```diff
@@ -0,0 +1,818 @@
+###############################################################################
+# WaterTAP Copyright (c) 2021, The Regents of the University of California,
+# through Lawrence Berkeley National Laboratory, Oak Ridge National
+# Laboratory, National Renewable Energy Laboratory, and National Energy
+# Technology Laboratory (subject to receipt of any required approvals from
+# the U.S. Dept. of Energy). All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and license
+# information, respectively. These files are also available online at the URL
+# "https://github.com/watertap-org/watertap/"
+#
+###############################################################################
+
+from copy import deepcopy
```

Comment:
`deepcopy`, `Enum`, and `auto` appear to be unused - can these import statements be removed?

Reply:
Removing those unused imports